app/models/manageiq/providers/openstack/network_manager.rb

```diff
@@ -147,7 +146,7 @@ def create_floating_ip_queue(userid, options = {})
     }
     queue_opts = {
       :class_name  => self.class.name,
-      :method_name => 'create_floating_ip',
+      :method_name => 'create_security_group',
```

Comment:
Is this change supposed to be here? 

Feels like `create_floating_ip_queue` should probably **not** call `create_security_group`...